### PR TITLE
Errors are bubbling and functions are named correctly

### PR DESCRIPTION
**Pull request by slvnperron**

### CODE REVIEW

File: lib/LambdaHelper.js [hunk 1]
```diff
@@ -68,7 +65,13 @@ var _lambdaize = function (userFunc) {
             func.apply(this, event.args);
         }
         catch(error) {
-            _newCallback({isFaulty: true, stack: error.stack, message: error.message});
+            var objectifyError = function(err) {
```
Comment (EFF) [inline lib/LambdaHelper.js]: Should this function be outside the catch block so it doesn't get instantiated every time there is an error ?

Comment (slvnperron) [inline lib/LambdaHelper.js]: No that is the purpose of it; there can be only one error therefore we
declare it only if it happens (very small optimization but why not)

File: lib/LambdaHelper.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ var _loadModule = function(moduleName) {
 
 var _lambdaize = function (userFunc) {
     function __lambda__ (event, context) {
+        var LambdaError = function() {};
         var executionSucceeded = true,
```
Comment (EFF) [inline lib/LambdaHelper.js]: these 2 vars necessary ? seem unused to me

**Comment by EFF** [comment]
LGTM after answering / fixing comments

**Pull request merged**